packages/PyTrilinos2/test/CMakeLists.txt

```diff
@@ -5,3 +5,21 @@ INCLUDE(PyTrilinos2MakeTest)
 
 PyTrilinos2_MAKE_MPI_TEST(CG)
 PyTrilinos2_MAKE_MPI_TEST(parameterList)
+
+TRIBITS_COPY_FILES_TO_BINARY_DIR(Stratimiko_cp
+    SOURCE_FILES Stratimikos.py)
+
+TRIBITS_ADD_TEST(
+  ${Python3_EXECUTABLE}
+  NOEXEPREFIX
+  NOEXESUFFIX
+  NAME Stratimikos
+  POSTFIX_AND_ARGS_0 "LU"                 Stratimikos.py --problemSize=100   --solver=LU
+  POSTFIX_AND_ARGS_1 "CG"                 Stratimikos.py --problemSize=100   --solver=CG       --prec=None
+  POSTFIX_AND_ARGS_2 "CG_Jacobi"          Stratimikos.py --problemSize=100   --solver=CG       --prec=Jacobi
+  POSTFIX_AND_ARGS_3 "BiCGStab_Chebyshev" Stratimikos.py --problemSize=100   --solver=BiCGStab --prec=Chebyshev
+  POSTFIX_AND_ARGS_4 "GMRES_ILU"          Stratimikos.py --problemSize=1000  --solver=GMRES    --prec=ILU
+  POSTFIX_AND_ARGS_5 "CG_multigrid"       Stratimikos.py --problemSize=10000 --solver=CG       --prec=multigrid
+  PASS_REGULAR_EXPRESSION "OK"
+  ENVIRONMENT "PYTHONPATH=${PyTrilinos2_PYTHONPATH}:$ENV{PYTHONPATH}"
```

Comment:
It is a bummer that we can't see this test running in PR testing.

Is there any plan to get PyTrilinos2 tested in PR testing?  It would seem that with the future of using Spack to create containers and build packages in those containers that Spack should be able to provide the needed dependencies (i..e. [Pybind11](https://packages.spack.io/package.html?name=py-pybind11) and [Binder](https://packages.spack.io/package.html?name=binder)).

Reply:
The framework team set up a nightly build for PyTrilinos2, but it's currently misconfigured. I think that the AT2 containers will all have pybind11 and binder. I am locally using a framework provided container for development.